roles/restore/tasks/init_awx.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+- name: Get AWX object definition from pvc
+  community.kubernetes.k8s_exec:
+    namespace: "{{ tower_backup_pvc_namespace }}"
+    pod: "{{ meta.name }}-db-management"
+    command: >-
+      bash -c "cat '{{ tower_backup_dir }}/awx_object.yml'"
+  register: awx_object
+
+- name: Write temp AWX definition template file
+  copy:
+    dest: "_definitions/awx_object.yml.j2"
+    content: |
+      {{ awx_object.stdout }}
+    mode: '0600'
+
+- name: Deploy AWX
+  k8s:
+    state: "{{ state | default('present') }}"
+    namespace: "{{ namespace | default('default') }}"
+    apply: yes
+    wait: yes
+    template: "_definitions/awx_object.yml.j2"
```

Comment:
Following along with my comment above, this could become something like:

```
definition: "{{ awx_object.stdout | from_yaml }}"
```

Reply:
When I tried this before, I used `definition: "{{ awx_object.stdout }}"`, maybe I just needed the `from_yaml`, I'll give that a try.

Reply:
@shanemcd passing yaml to a `definition:` field like that works with `from_yaml`, but it won't template the `name` and `namespace` fields.  So we'll still need to template to a file.